grassroots-backend/src/grassroots-shared/Contact.entity.dto.ts

```diff
@@ -75,23 +78,34 @@ export class ContactSearchInDTO {
   @IsInt()
   @Min(1)
   id?: number;
+  @IsOptional()
   email?: string;
+  @IsOptional()
   firstName?: string;
+  @IsOptional()
   lastName?: string;
+  @IsOptional()
```

Comment:
Is this required in NestJS if there's a `?`?

Reply:
Yeah. There are two ways Nest uses "reflection".

1. Reflect-metadata, which is just about inspecting decorators and their parameters.
2. The nest openAPI cli plug-in, which does a very limited set of things 

Neither of those handle this case. The nest eslint plug-in catches this, making it a bit less annoying.